cloudify_ansible_sdk/sources.py

```diff
@@ -18,28 +18,52 @@ class AnsibleSource(object):
     def __init__(self, source_dict):
 
         source_dict = source_dict or {}
-        self.groups = {}
+        self.children = {}
+        self.hosts = {}
+
+        if 'all' in source_dict:
+            self.do_insert_children(source_dict['all'].get('children', {}))
```

Comment:
I think since source_dict could be {} we need to make sure that KeyError will no raise here , what do you think ?